RecoEgamma/EgammaElectronAlgos/src/GsfElectronAlgo.cc

```diff
@@ -1114,17 +1114,15 @@ bool GsfElectronAlgo::isPreselected( GsfElectron * ele )
  {
 	bool passCutBased=ele->passingCutBasedPreselection();
 	bool passPF=ele->passingPflowPreselection(); //it is worth nothing for gedGsfElectrons, this does nothing as its not set till GedGsfElectron finaliser, this is always false
-	if(generalData_->strategyCfg.gedElectronMode){
+        if(generalData_->strategyCfg.gedElectronMode){
```

Comment:
indentation

Reply:
(you may need to untabify)
usually a local check with ```git diff ``` in the terminal is enough to spot these inconsistencies before pushing to the remote

Reply:
thx, i have fixed this